lock/src/main/java/org/infinispan/lock/api/impl/ClusteredLockModuleLifecycle.java

```diff
@@ -0,0 +1,103 @@
+package org.infinispan.lock.api.impl;
+
+import java.util.EnumSet;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+import org.infinispan.Cache;
+import org.infinispan.commons.marshall.AdvancedExternalizer;
+import org.infinispan.configuration.cache.CacheMode;
+import org.infinispan.configuration.cache.Configuration;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.configuration.global.GlobalConfiguration;
+import org.infinispan.factories.GlobalComponentRegistry;
+import org.infinispan.lifecycle.AbstractModuleLifecycle;
+import org.infinispan.lifecycle.ModuleLifecycle;
+import org.infinispan.lock.api.ClusteredLockManager;
+import org.infinispan.lock.api.impl.entries.ClusteredLockKey;
+import org.infinispan.lock.api.impl.entries.ClusteredLockState;
+import org.infinispan.lock.api.impl.entries.ClusteredLockValue;
+import org.infinispan.lock.api.impl.functions.IsLocked;
+import org.infinispan.lock.api.impl.functions.LockFunction;
+import org.infinispan.lock.api.impl.functions.UnlockFunction;
+import org.infinispan.lock.api.impl.lock.ClusteredLockFilter;
+import org.infinispan.lock.api.impl.manager.CacheHolder;
+import org.infinispan.lock.api.impl.manager.EmbeddedClusteredLockManager;
+import org.infinispan.manager.EmbeddedCacheManager;
+import org.infinispan.partitionhandling.PartitionHandling;
+import org.infinispan.registry.InternalCacheRegistry;
+import org.infinispan.transaction.TransactionMode;
+import org.kohsuke.MetaInfServices;
+
+/**
+ * Locks module configuration
+ *
+ * @author Katia Aresti, karesti@redhat.com
+ * @since 9.2
+ */
+@MetaInfServices(value = ModuleLifecycle.class)
+public class ClusteredLockModuleLifecycle extends AbstractModuleLifecycle {
```

Comment:
`AbstractModuleLifecycle` is deprecated! use `ModuleLifecycle` instead.

Reply:
ok